src/ngx_pagespeed.cc

```diff
@@ -749,6 +749,11 @@ ngx_command_t ps_commands[] = {
   ngx_null_command
 };
 
+bool ps_disabled(ps_srv_conf_t* cfg_s) {
```

Comment:
is this in an anon namespace?  I briefly expanded the whole file and then decided to just ask :)